.github/workflows-source/release-testing.yml

```diff
@@ -46,44 +46,29 @@ jobs:
     uses: ./.github/workflows/ci-main.yml
     secrets: inherit
 
-  bazel-system-test-nightly:
-    name: Bazel System Test Nightly
+  # Run some extra system tests that are skipped on CI Main.
+  # Those test suites would overwhelm a runner, so each is run
+  # as a separate job.
+  bazel-system-tests:
+    name: Bazel System Tests (${{ matrix.tag }})
     <<: *dind-large-setup
+    strategy:
+      matrix:
+        tag:
+          - system_test_nightly
+          - system_test_staging
+          - system_test_hotfix
     steps:
       - <<: *checkout
-      - name: Run Bazel System Test Nightly
-        id: bazel-test-all
-        uses: ./.github/actions/bazel-test-all/
-        with:
-          BAZEL_COMMAND: test --keep_going --test_tag_filters=system_test_nightly
-          BAZEL_TARGETS: //rs/tests/...
-          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-
-  bazel-system-test-staging:
-    name: Bazel System Test Staging
-    <<: *dind-large-setup
-    steps:
-      - <<: *checkout
-      - name: Run Bazel System Test Staging
-        id: bazel-test-all
-        uses: ./.github/actions/bazel-test-all/
-        with:
-          BAZEL_COMMAND: test --keep_going --test_tag_filters=system_test_staging
-          BAZEL_TARGETS: //rs/tests/...
-          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-
-  bazel-system-test-hotfix:
-    name: Bazel System Test Hotfix
-    <<: *dind-large-setup
-    timeout-minutes: 90
```

Comment:
Wasn't clear why this was set, so I dropped it. Happy to add it back if necessary